public/maps.js

```diff
@@ -46,6 +47,25 @@ const FourG2017 = [
   { code: 'sa', regionName: "Latin America & Caribbean", value: 82 }
 ];
 
+// 5G action plans -http://www.europarl.europa.eu/RegData/etudes/IDAN/2019/631060/IPOL_IDA(2019)631060_EN.pdf
+const FiveG2025 = [
+  { code: 'USA', regionName: "USA" },
+  { code: 'TWN', regionName: "Taiwan" },
+  { code: 'SGP', regionName: "Singapore" },
+  { code: 'KOR', regionName: "Korea" },
+  { code: 'JPN', regionName: "Japan" },
+  { code: 'CHN', regionName: "China" },
+  { code: 'AUT', regionName: "Austria" },
+  { code: 'FRA', regionName: "France" },
+  { code: 'FIN', regionName: "Finland" },
+  { code: 'NLD', regionName: "Netherlands" },
+  { code: 'ESP', regionName: "Spain" },
+  { code: 'SWE', regionName: "Sweden" },
+  { code: 'DEU', regionName: "Germany" },
+  { code: 'LUX', regionName: "Luxemburg" },
+  { code: 'GBR', regionName: "UK" }
+];
```

Comment:
still have to generate a map from this

Reply:
if we don't get to this, I think that's fine